index.bs

```diff
@@ -2203,6 +2210,14 @@ browsingContext.InnerTextLocator = {
    ? maxDepth: js-uint,
 }
 
+browsingContext.AccessibilityLocator = {
+   type: "accessibility",
+   value: {
+    ? role: text,
+    ? name: text,
```

Comment:
```suggestion
    ? name: text,
    ? role: text,
```

Beside that one more question... In WebDriver classic we have [`Get Computed Label`](https://w3c.github.io/webdriver/#get-computed-label), so should we use `label` instead of `name`?

Reply:
I am not sure why it is called label since it references accessible name in the spec (the spec name is "Accessible Name and Description Computation 1.1")

Reply:
I can see: 

> The purpose of the computation is to create a [perceivable](https://www.w3.org/TR/accname-1.1/#dfn-perceivable) label or description for alternative presentations, in the form of a flat string of space separated textual tokens.

@jgraham what's your take on that?

Reply:
I would be in the favor of name as well, since this is about accessible name. The accessible name might be derived from an aria-label, but not necessarily (can come from title etc...).
Also, both Firefox and Chrome DevTools expose this as "name" in the UI, so it could be helpful for users writing tests if it matches our APIs.

Reply:
Ah good to know. Aligning what DevTools has might actually be also a good idea and will help web developers and testers to align